[GHES] Added manifest for Slack app

aasiddiq opened the pull request:
Added the manifest for slack for app creation to support GHES

ashokirla reviewed (changes requested):
@aasiddiq Teams manifest should be in integrations/ms-teams repo. We will also breakdown the steps for each of them separate and put them in both repos respectively.

aasiddiq commented:
> @aasiddiq Teams manifest should be in integrations/ms-teams repo. We will also breakdown the steps for each of them separate and put them in both repos respectively.

Removed the team manifest and added in https://github.com/integrations/microsoft-teams/pull/226
